packages/cli/src/backend-identifier/backend_identifier_with_sandbox_fallback.ts

```diff
@@ -18,23 +18,25 @@ export class BackendIdentifierResolverWithFallback
     private fallbackResolver: SandboxBackendIdResolver
   ) {}
   /**
-   * resolves the backend id, falling back to the sandbox id if there is an error
+   * resolves to deployed backend id, falling back to the sandbox id if there is an error and stack, appId, and branch are not provided
```

Comment:
We are not falling back on errors

```suggestion
   * resolves to deployed backend id, falling back to the sandbox id if stack or appId and branch inputs are not provided
```